app/src/main/AndroidManifest.xml

```diff
@@ -5,7 +5,10 @@
 
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
I guess this is to check if the user is connected to internet or not 👍

Reply:
That's right. But in the end, I might not use this way of checking the internet connection since we can simply a ping command at runtime to some website (e.g google.com). This might be an easier way to do it, so I might simply delete this line.